### ISSUE TITLE
Unable to launch local web page (:10000)

### ISSUE THREAD

**Issue by thebaldgeek** [ahupowerdns/galmon#15]
Raspberry Pi 3+.  Buster. Ublox8.

`pi@raspberrypi:~/galmon $ ./ubxtool --wait --port /dev/ttyACM0 --station 1 --stdout | ./ubxparse 10000 html null

-bash: ./ubxparse: No such file or directory
Sending version query
swVersion: ROM CORE 3.01 (107888)
hwVersion: 00080000
Extended info: FWVER=SPG 3.01
Extended info: PROTVER=18.00
Extended info: GPS;GLO;GAL;BDS
Extended info: SBAS;IMES;QZSS
Sending GNSS query
GNSS status, got 7 rows:
GNSSID 0 enabled 1 minTrk 8 maxTrk 16 1 0  1 1
GNSSID 1 enabled 1 minTrk 1 maxTrk 3 1 0  1 1
GNSSID 2 enabled 0 minTrk 4 maxTrk 8 0 0  1 1
GNSSID 3 enabled 0 minTrk 8 maxTrk 16 0 0  1 1
GNSSID 4 enabled 0 minTrk 0 maxTrk 8 0 0  1 3
GNSSID 5 enabled 1 minTrk 0 maxTrk 3 1 0  1 5
GNSSID 6 enabled 1 minTrk 8 maxTrk 14 1 0  1 1
Got an ack for 6 62 while waiting for 6 62
Got ACK for our poll of GNSS settings
Sending GNSS setting, GPS: 0, Galileo: 0, BeiDou: 0, GLONASS: 0, SBAS: 0
Got an nack for 6 62 while waiting for 6 62
Got nack on GNSS setting
pi@raspberrypi:~/galmon $`

At this point, I visit my pi IP address:10000 and can not connect.

Any assistance would be gratefully received.

**Comment by thebaldgeek** [comment]
I created the file 'ubxparse' and that has removed that error, but still can not connect to localhost:10000

**Comment by thebaldgeek** [comment]
Tried to 'make ubxparse' Here is the resulting error; https://pastebin.com/JZgzNmg4

**Comment by thebaldgeek** [comment]
I am up and running.
The issue was two fold, firstly, not all satellite types are supported by my ublox8 and so the feed crashed with a simple 'bad magic'. But using the other command line options to nominate which sats you want I was able to determine that the beidou option was causing the crash.

The second issue was the navparse option for simply port 10000 was causing my Pi to listen on an odd IP address, at first I tried 127.0.0.1:10000 but even that would not start the web server, I had to put the IP address of my pi, so the full command line becomes;
  ./ubxtool --wait --port /dev/ttyACM0 --gps --glonass --galileo --sbas --station 22 --stdout | ./navparse 192.168.1.240:10000 html null
